organization/update.ts

```diff
@@ -1,96 +1,79 @@
-import * as gracely from "gracely"
-import * as model from "@userwidgets/model"
-import * as http from "cloudly-http"
+import { gracely } from "gracely"
+import { isoly } from "isoly"
+import { userwidgets } from "@userwidgets/model"
+import { http } from "cloudly-http"
 import { common } from "../common"
 import { Context } from "../Context"
 import { router } from "../router"
 
+type Response =
```

Comment:
This response type is horrifying but a necessary evil until webhooks are implemented for events like this.